imports/api/activities/activity.js

```diff
@@ -59,8 +53,7 @@ Schemas.Activity = new SimpleSchema({
 
   isSentForReview: {type: Boolean},
   isPublicActivity: {type: Boolean, optional: true},
-  // isBookingsDisabled: {type: Boolean, optional: true}, //undefined
-  isActivitiesDisabled: {type: Boolean, optional: true},
+  isRegistrationDisabled: {type: Boolean, optional: true}, //undefined
```

Comment:
this change can cause missing data: for whom used this feat before stored their data under isActivitiesDiasabled key so for those isRegistrationDisabled will return undefined or else.
We need to do migration for these kind of changes.
https://guide.meteor.com/collections.html#writing-migrations

Reply:
This was falsely written as isActivitiesDisabled, and I wanted to change that... We need to, cause it's semantically wrong with isActivities (isBookings is OK but better with isRegistration...)

Lets consider how we can do the migrations but we can run a few lines script also I think...